testsuite/classlibrary/TestServer_clientIDBooted.sc

```diff
@@ -0,0 +1,27 @@
+// see also TestServer_clientID
+TestServer_clientID_booted : UnitTest {
+
+	// with running server
+	test_ClientIDResetByServer {
+		var options = ServerOptions.new;
+		var s;
+		s = Server(\testserv1, NetAddr("localhost", 57111), options);
+
+		s.options.maxLogins = 4; s.clientID = 3;
```

Comment:
lines 10, 15, 25 - one statement per line please

Reply:
still 2 statements on this line

Reply:
still 2 statements on this line @adcxyz

Reply:
done.